docker/ngi_config.yaml

```diff
@@ -0,0 +1,127 @@
+# This file should be placed under $HOME/.ngiconfig/ngi_config.yaml
+# or its path exported as the environment variable NGI_CONFIG
+
+database:
+    record_tracking_db_path: /data/ngi2016000/private/db/records_db.sql
+
+environment:
+    project_id: ngi2016000
+    ngi_scripts_dir: /ngi_pipeline/scripts 
+    conda_env: NGI
+    flowcell_inbox:
+            - /data/ngi2016000/incoming
+ 
+piper:
+    #sample:
+    #    required_autosomal_coverage: 28.4
+    load_modules:
+        - bioinfo-tools piper/1.5.1
+    threads: 16
+    job_walltime:
+        merge_process_variantcall: "10-00:00:00"
+    shell_jobrunner: Shell
+    #shell_jobrunner: ParallelShell --super_charge --ways_to_split 4
+    #jobNative:
+    #    - arg1
+    #    - arg2
+    #    - arg3
+    # TODO: Piper module adds setupfilecreator to the path; unsure if this works or if complete path is required.  
+    path_to_setupfilecreator: setupfilecreator
+    gatk_key: "/path/to/gatk.key"
+
+sarek:
+    tag: 2.6
```

Comment:
Sarek 2.6.1 has now been released with GATK updated. Version 2.6 didn't work completely when we tried to run it on Irma.

Reply:
thanks, I'll update this config and also the upcoming PR to irma-provision when this has been merged